webdav/server.go

```diff
@@ -43,14 +43,14 @@ func main() {
 	)
 
 	flag.IntVar(&port, "port", 8079, "this server's port")
-	flag.StringVar(&proxy, "proxyurl", "127.0.0.1:8080", "dfc proxy's url (ip:port)")
-	flag.StringVar(&tmpDir, "tmpdir", "/tmp/dfc", "temporary directory to store files used by webdav")
-	flag.IntVar(&logLevel, "loglevel", logLevelNone, "verbose level(0 = none, 1 = webdav, 2 = dfc)")
+	flag.StringVar(&proxy, "proxyURL", "127.0.0.1:8080", "dfc proxy's URL (ip:port)")
```

Comment:
this is suppose to be proxy url, this is the command line option's shown name, why change to proxyURL? same for tmpDir and LogLevel